python/huggingfaceserver/tests/test_vlllm_model.py

```diff
@@ -0,0 +1,2898 @@
+        expected.choices[0].logprobs.token_logprobs = [
+            None,
+            -9.352765083312988,
+            -1.4278249740600586,
+            -0.976689338684082,
+            -5.6148481369018555,
+            -4.214991569519043,
+            -4.99854040145874,
+            -5.968787670135498,
+            -11.009231567382812,
+            -3.1531941890716553,
+            -1.4167277812957764,
+            -2.766524314880371,
+            -6.9396467208862305,
+            -1.3619931936264038,
+            -4.619960308074951,
+            -5.6248779296875,
+            -2.2152767181396484,
+        ]
+        # FixMe: pydantic does not allows adding None to the top_logrobs list. We should fix the type definition.
+        expected.choices[0].logprobs.top_logprobs = [
+            None,
+            {
+                "Hi": -9.352765083312988,
+                "I": -1.4278708696365356,
+                "The": -2.4365129470825195,
+            },
+            {",": -1.4278249740600586, "!": -1.934173583984375},
+            {" I": -0.976689338684082, " ": -2.723400115966797},
+            {
+                " love": -5.6148481369018555,
+                "'m": -1.015452265739441,
+                " have": -1.9374703168869019,
+            },
+            {
+                " my": -4.214991569519043,
+                " your": -1.7619359493255615,
+                " the": -1.999145269393921,
+            },
+            {
+                " cat": -4.99854040145874,
+                " new": -3.4642574787139893,
+                " old": -4.73804235458374,
+            },
+            {"-": -5.968787670135498, ".": -1.453755497932434, ",": -1.841694951057434},
+            {
+                " Labrador": -11.009231567382812,
+                " I": -1.754422903060913,
+                " she": -3.075488328933716,
+            },
+            {
+                "!": -3.1531941890716553,
+                " mix": -1.0394361019134521,
+                ".": -2.1872146129608154,
+            },
+            {" He": -1.4167277812957764, "\n": -2.0672662258148193},
+            {
+                " has": -2.766524314880371,
+                "'s": -1.0847474336624146,
+                " is": -1.547521710395813,
+            },
+            {
+                " tiny": -6.9396467208862305,
+                " a": -1.3877270221710205,
+                " been": -2.3109371662139893,
+            },
+            {" ears": -1.3619931936264038, " paws": -2.2743258476257324},
+            {
+                " with": -4.619960308074951,
+                " and": -0.805719792842865,
+                ",": -1.6155686378479004,
+            },
+            {
+                " fluffy": -5.6248779296875,
+                " a": -1.4977400302886963,
+                " tiny": -3.006150484085083,
+            },
+            {" white": -2.2152767181396484, " fur": -1.9012728929519653},
+        ]
+        assert response.id == expected.id
```

Comment:
Should we have these comparisons as a function and do 
`assert compare_response_to_expected(response,expected)`
```
def compare_response_to_expected(actual,expected, fields_to_compare=[default_fields]) -> bool:
   for field in fields_to_compare:
      if actual.field!=expected.field: 
       return False
  return True
```
I see that we're using this at multiple places